python-checks/src/test/resources/checks/tests/implicitlySkippedTestWithWrapper.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
It looks like this directly comes from cases found on ruling.

In general, it is better to try and reduce those cases a bit to have more minimal test cases.
Some of the imports could probably be removed (for example `runtime` don't feel strictly necessary for the test - any random `cond()` could replace the use of `runtime.platformType` for instance).

Despite `twisted.trial.unittest` surely being a thing, I guess we don't actually resolve this and we could replace it by some random and obviously fake import just to highlight that what we do is look for `unittest` and `TestCase` in the name without doing a full resolution.

We could probably also remove the mixin, as I guess it doesn't impact the result either (also multiple inheritance is already tested somewhere else).